Lombiq.NodeJs.Extensions/scripts/run-pnpm-script.js

```diff
@@ -0,0 +1,70 @@
+﻿const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const { handleErrorMessage } = require('./handle-error');
+
+function panic(message) {
+    handleErrorMessage(message);
+    process.exit(1);
+}
+
+// Check if pnpm is installed.
+try { child_process.execSync('pnpm -v'); }
+catch (_) { panic('This project requires pnpm. You can install it with the "npm install --global pnpm" command.'); }
+
+// Load command line arguments.
+const args = process.argv.splice(2);
+if (args.length < 2) panic("USAGE: node scripts/run-pnpm-script project-path script")
+
+// Initialize variables.
+const [ projectPath, script ] = args;
+const packageJsonPath = path.join(projectPath, 'package.json');
+
+function main() {
+    // Create the package.json file if it doesn't exist.
+    if (!fs.existsSync(packageJsonPath)) {
+        fs.copyFileSync(
+            path.resolve('..', 'config', 'consumer', 'package.project.json'),
+            packageJsonPath);
+    }
+
+    // Go to project directory and read the package.json file to find the scripts.
+    process.chdir(projectPath);
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, { encoding: 'utf8' }))
+    const scripts = packageJson.scripts ?? { };
+
+    // Handle copy:assets which is a special case during build/compile.
+    if ((script === 'build' || script === 'compile') && 'assetsToCopy' in packageJson) {
```

Comment:
If [OSOE-84](https://lombiq.atlassian.net/browse/OSOE-84) is merged, this is the necessary change for the updated _package.json_ structure.
```suggestion
    if ((script === 'build' || script === 'compile') &&
        'nodejsExtensions' in packageJson &&
        'assetsToCopy' in packageJson.nodejsExtensions) {
```

Reply:
Thanks. My plan was to merge OSOE-92 - either into `task/node-extensions` or `issue/OSOE-84` - first, but let's see.

Reply:
Sure. If this is merged first, then the above change has to be done in OSOE-92 so refer to this.